dvc/exceptions.py

```diff
@@ -349,3 +349,9 @@ def __init__(self, dep, a, b):
 class PrettyDvcException(DvcException):
     def __pretty_exc__(self, **kwargs):
         """Print prettier exception message."""
+
+
+class GlobDoesNotMatchError(DvcException):
+    def __init__(self, glob):
+        msg = f"Glob {glob} has no matches."
+        super().__init__(msg)
```

Comment:
Let's keep the exception in `dvc/utils/__init__.py`, closer to where it is used.

Reply:
Importing `DvcException` at top for custom exception raises circular import error. I have used `DvcException` itself.